firedrake/preconditioners/pmg.py

```diff
@@ -32,7 +32,7 @@ class PMGBase(PCSNESBase):
     Other PETSc options inspected by this class are:
     - 'pmg_coarse_degree': polynomial degree of the coarse level
     - 'pmg_coarse_mat_type': can be either 'aij' or 'matfree'
-    - 'pmg_coarse_form_compiler_mode': can be 'spectral' (default), 'vanilla', 'coffee', or 'tensor'
+    - 'pmg_coarse_form_compiler_mode': can be 'spectral' (default), 'vanilla', or 'tensor'
```

Comment:
Revert? Rename "legacy"??

Reply:
Reverted for now. I appreciate that "coffee" is no longer a very sensible name for a TSFC mode.